src/ring_theory/ideal_operations.lean

```diff
@@ -680,6 +747,34 @@ theorem mem_jacobson_iff {I : ideal R} {x : R} :
         neg_mul_eq_neg_mul_symm, neg_mul_eq_mul_neg, mul_comm x y]; exact M.mul_mem_right hy)
     (him hz)⟩
 
+lemma jacobson_mono {I J : ideal R} : I ≤ J → I.jacobson ≤ J.jacobson :=
+begin
+  intros h x hx,
+  erw mem_Inf at ⊢ hx,
+  exact λ K ⟨hK, hK_max⟩, hx ⟨trans h hK, hK_max⟩
+end
+
+lemma comap_jacobson {S : Type u} [comm_ring S] (e : S ≃+* R) {I : ideal R}
+    : comap e.to_ring_hom (jacobson I) = jacobson (comap e.to_ring_hom I) :=
```

Comment:
You shouldn't need to write the `.to_ring_hom`. If you do, we need to fix this, by adding a `coe` somewhere.

Reply:
There currently isn't a `coe` from `ring_equiv` to `ring_hom`. I tried adding one in, and then I am able to replace `e.to_ring_hom` on the LHS with `(e : S →+* R)`, but I can't put just e on its own. I can also replace the RHS with just `↑e`, but I can't put `↑e` on both sides, just the RHS. 

Is this generally what would be expected from adding in the `coe` instance? It sort of makes sense to me that the type checker would struggle still since the signature doesn't explicitly contain the types of the underlying rings.